providers/openshift/provider.go

```diff
@@ -308,6 +314,14 @@ func (p *OpenShiftProvider) Complete(data *providers.ProviderData, reviewURL *ur
 	p.ProviderData = data
 	p.ReviewURL = reviewURL
 
+	// get anonymous kubeconfig to copy for client calls
+	config, err := rest.InClusterConfig()
```

Comment:
no.  YOu need to support flags, like this https://github.com/deads2k/oauth-proxy/blob/60455f2656700d473797bce5315253cc5ceb3035/providers/openshift/authentication.go#L193-L194